File: cpp/bc1192.cpp
#include <iostream>

using namespace std;

int main()
{
	int fVal, sVal, res, nCa;
	char le;
	
	cin >> nCa;
	
	for(int i = 0; i < nCa; i++)
	{
		cin >> fVal >> le >> sVal;
		
		if(fVal == sVal) res = fVal * sVal;
		else res = (int) le < 97? sVal - fVal : fVal + sVal;
		
		cout << res << endl;
	}
}

File: python/bc1192.py
nCa = int(input())

for c in range(0, nCa):
    cVal = input()

    fVal = int(cVal[0])
    le = cVal[1]
    sVal = int(cVal[2])

    if fVal == sVal: res = fVal * sVal
    else: res = sVal - fVal if le.isupper() else fVal + sVal

    print(res)